dashboard/src/components/AppView/AppView.tsx

```diff
@@ -148,7 +148,15 @@ class AppView extends React.Component<IAppViewProps, IAppViewState> {
 
   public render() {
     if (this.props.error) {
-      return this.renderError(this.props.error);
+      return (
+        <ErrorSelector
+          error={this.props.error}
+          defaultRequiredRBACRoles={RequiredRBACRoles}
```

Comment:
Why is this called `defaultRequiredRBACRoles`? Why default? Wouldn't `requiredRBACRoles` make more sense?